tests/functiontest/conftest.py

```diff
@@ -321,7 +316,7 @@ def reportinfo(self):
         about the test case. The third tuple item is a string that
         identifies the test case in a human readable way.
         """
-        return self.fspath, 0, "%s in %s" % (self.name, self.filepath)
+        return self.fspath, 0, "{} in {}".format(self.name, self.filepath)
```

Comment:
f" formatter ?

Reply:
Yes. pyupgrade changed this when re-run, to use f-strings.